kasa/experimental/modules/led.py

```diff
@@ -0,0 +1,28 @@
+"""Module for led controls."""
+
+from __future__ import annotations
+
+from ...interfaces.led import Led as LedInterface
```

Comment:
I'm wondering if we should get rid of the led interface at some point, but it's something for later.

Reply:
It’s my favourite interface. So simple 😄